CONTRIBUTING.md

```diff
@@ -10,6 +10,8 @@ There are several ways to contribute to the project:
 
 GitHub supports [markdown](http://github.github.com/github-flavored-markdown/), so when filing bugs make sure you check the formatting before clicking submit.
 
+ews-java-api uses findbugs to check the code, and you should do so too before submitting your code. A findbugs report can be generated using "mvn site", or by downloading it as a plugin for your favorite IDE.
```

Comment:
maybe this should be added to an independent section as there might join more suggested plugins